src/Platforms/PostgreSQLPlatform.php

```diff
@@ -833,6 +833,12 @@ public function getDropIndexSQL($index, $table = null)
             return $this->getDropConstraintSQL($constraintName, $table);
         }
 
+        $tableName = $table instanceof Table ? $table->getQuotedName($this) : $table;
+        if ($tableName !== null && strpos($tableName, '.') !== false) {
+            [$schema] = explode('.', $tableName);
+            $index    = $schema . '.' . ($index instanceof Index ? $index->getQuotedName($this) : $index);
```

Comment:
Passing `$index` as an `Index` object to `getDropIndexSQL` was deprecated and we need to convert it to a quoted name. I don't think it can be replicated in a functional test.

Reply:
The "not covered by tests" warning is likely a false positive. According to Codecov, the PR has 100% diff coverage, and this lie is [marked as covered](https://app.codecov.io/gh/doctrine/dbal/pull/6779?dropdown=coverage&src=pr&el=h1#64a390a1143276824549dc48f99b1f8a-R839) on Codecov. Maybe the warnings are produced using more strict logic than the coverage report (which requires both branches of the ternary operator to be covered), but this indeed can be ignored.